client/state/jetpack-onboarding/selectors.js

```diff
@@ -0,0 +1,23 @@
+/** @format */
```

Comment:
Any reason not to place this selector in the global selectors directory?

Reply:
My old 'but I want them modularized' pet peeve. I can go with the majority agreement tho...

Reply:
I always introduce new selectors to the global dir, but I'll leave that to your judgement 😉

Reply:
Yeah, I'll move it over.